test/auditing.jl

```diff
@@ -27,7 +27,7 @@ end
     @test compatible_marchs(Platform("x86_64", "linux"; march="avx2")) == ["x86_64", "avx", "avx2"]
     @test compatible_marchs(Platform("x86_64", "linux"; march="avx512")) == ["x86_64", "avx", "avx2", "avx512"]
     @test compatible_marchs(Platform("armv7l", "linux")) == ["armv7l"]
-    @test compatible_marchs(Platform("i686", "linux"; march="prescott")) == ["i686", "prescott"]
+    @test compatible_marchs(Platform("i686", "linux"; march="prescott")) == ["pentium4", "prescott"]
```

Comment:
I _think_ the version of Julia we have here requires `i686`, with Julia v1.6 we can do `pentium4`

Reply:
Good catches all, this is meant to be merged once we switch over to v1.6.